openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -50,7 +50,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |auth_crypt                         | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|auth_ldap                          |                                   |
+|auth_ldap                          | No change                         |
```

Comment:
It says on the top of [the file](https://github.com/OCA/OpenUpgrade/blob/9.0/openerp/openupgrade/doc/source/modules80-90.rst) that
> 'No change' : No change between 8.0 and 9.0;
> 'Nothing to do' : Analyse has been done, and there is nothing to do;

If it's a redundant status, I suggest we remove it.